Lab_6.ipynb

```diff
@@ -0,0 +1,555 @@
+      "text/plain": [
+       "<Figure size 576x396 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1,1)\n",
+    "ax.hist(background, bins=30, density=True, label='Background')\n",
+    "plt.axvline(threshold_value, color='k', linestyle='--', label=\"Threshold value: \"+str(threshold_value))\n",
+    "ax.legend(loc=0)\n",
+    "plt.tick_params(labelsize = 13)\n",
+    "plt.show()"
```

Comment:
May want to also print the value so that it's easier to see, aside from just being on the plot